src/platform/tests/PlatformTests.cpp

```diff
@@ -26,12 +26,7 @@ namespace Tests
         }
         else
         {
-            char* handle = new (std::nothrow) char[strlen(g_mockHandle) + 1];
-            if (handle != nullptr)
-            {
-                strcpy(handle, g_mockHandle);
-            }
-            return (MPI_HANDLE)handle;
+            return (MPI_HANDLE)strdup(g_mockHandle);
```

Comment:
>strdup [](http://example.com/codeflow?start=31&length=6)

Even better, use our own DuplicateString (which today is same as strdup for a constant string, but in the future may gain more abilities).

Reply:
I'm aware of DuplicateString, but used the builtin here because it's only a tests and I don't see much point in using wrapper here.

Reply:
Thanks for your explanation! Our test code should make use of our internal common APIs as much as possible, to ensure greatest coverage. It's not a big deal, but please consider replacing this. If you decide to keep built-in strdup, that's fine with me. Please proceed whichever way you find more appropriate.

Reply:
Keeping strdup here as discussed.